src/external/wannierio.jl

```diff
@@ -0,0 +1,693 @@
+using Dates
+using Printf: @sprintf
+using SpecialFunctions: erfc
+
```

Comment:
You generally use far too restrictive type bounds. Let's discuss this in person @qiaojunfeng.